public/docs/ts/latest/guide/template-syntax.jade

```diff
@@ -778,7 +775,7 @@ block style-property-name-dart-diff
 
   <a id="eventemitter"></a>
   <a id="custom-event"></a>
-  ### Custom Events with EventEmitter
+  ### Custom events with EventEmitter
```

Comment:
Please make "EventEmitter" italic. That's the (desired) standard presentation of code elements in a header ... because `EventEmitter` doesn't display well in a header.